src/transformers/modeling_tf_utils.py

```diff
@@ -2863,7 +2853,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
 
         safetensors_from_pt = False
         if filename == SAFE_WEIGHTS_NAME:
-            with safe_open(resolved_archive_file, framework="tf") as f:
+            with safe_open(resolved_archive_file, framework="np") as f:
```

Comment:
Does this have any impact? 

keeping everything `tf` in a `tf` files seems easier imho.